src/Screens/ConfirmationScreen/confirmationPage.tsx

```diff
@@ -1,10 +1,233 @@
+                <Image
+                  source={{
+                    uri: item.dish.imageURL,
+                  }}
+                  style={confirmationPageStyles.imageStyle}
+                />
+                <View style={confirmationPageStyles.quantityContainer}>
+                  <Text style={confirmationPageStyles.quantityItem}>
+                    X{item.quantity}
+                  </Text>
+                </View>
+              </View>
+              <View style={confirmationPageStyles.textItemsContainer}>
+                <View>
+                  <Text style={confirmationPageStyles.itemTitle}>
+                    {item.dish.name}
+                  </Text>
+                </View>
+                {excludedIngredients[index]?.length > 0 ? (
+                  <View style={confirmationPageStyles.excludedIngWrapper}>
+                    <Text style={confirmationPageStyles.withoutIngText}>
+                      Без добавления:
+                    </Text>
+                    <Text style={confirmationPageStyles.ingTextWrapper}>
+                      {excludedIngredients[index]?.join(", ")}
+                    </Text>
+                  </View>
+                ) : null}
+                <View style={confirmationPageStyles.priceAndIconsContainer}>
+                  <Text style={confirmationPageStyles.priceItem}>
+                    {item.dish.price} BYN
+                  </Text>
+                  <Text style={confirmationPageStyles.priceItem2}>
+                    за 1 позицию
+                  </Text>
+                </View>
+              </View>
+            </View>
+          ))}
+          <View style={confirmationPageStyles.lineView}></View>
+          <View style={confirmationPageStyles.itemsWrapperContainer}>
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Тип заказа:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.orderType}
+              </Text>
+            </View>
+            {orderData.adress ? (
+              <View style={confirmationPageStyles.itemWrapperStyle}>
+                <Text style={confirmationPageStyles.firstPartTextStyle}>
+                  Адрес:
+                </Text>
+                <Text
+                  style={confirmationPageStyles.secondPartTextStyle}
+                  key={uuid.v4()}
+                >
+                  {formatAdress(orderData.adress)}
+                </Text>
+              </View>
+            ) : null}
+            <View style={confirmationPageStyles.itemWrapperStyle}>
+              <Text style={confirmationPageStyles.firstPartTextStyle}>
+                Дата:
+              </Text>
+              <Text style={confirmationPageStyles.secondPartTextStyle}>
+                {orderData.date}
+              </Text>
+            </View>
```

Comment:
move to the common component and reuse after

Reply:
fixed